js/components/countDownClock.js

```diff
@@ -198,7 +198,24 @@ CountDownClock.propTypes = {
 CountDownClock.defaultProps = {
   timeToShow: "10", //seconds
   clockWidth: 36,
-  currentPlayhead: 0
+  currentPlayhead: 0,
```

Comment:
JSDOM does not recognize the `<canvas>` element used in this component so the test is still blank